ReactWindows/ReactNative.Net46.Tests/Bridge/Queue/MessageQueueThreadTests.cs

```diff
@@ -34,7 +34,7 @@ public void MessageQueueThread_CreateUiThread_ThrowsNotSupported()
             );
         }
 
-        [SetUp]
+        // ToDo: Conflicts with UIManagerModule tests [SetUp]
         public void SetUp()
```

Comment:
>SetUp [](start = 20, length = 5)

Why don't you just create a singleton class for initializing the Application and expose a method called `Initialize` that is idempotent?